src/backends/github.rs

```diff
@@ -340,9 +346,10 @@ impl UpdateBuilder {
 
     /// Set the exe's name. Also sets `bin_path_in_archive` if it hasn't already been set.
```

Comment:
mind adding a mention of bin name requirements for windows in this method's docs?